src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -737,6 +758,33 @@ void StridedSlice::StridedSliceCommonExecutor::exec(const std::vector<MemoryCPtr
     });
 }
 
+void StridedSlice::StridedSliceCommonExecutor::execSliceScatter(const std::vector<MemoryCPtr>& srcMemory, const std::vector<MemoryCPtr>& dstMemory) {
+    const uint8_t* srcData = srcMemory[0]->getDataAs<const uint8_t>();
+    const uint8_t* srcUpdates = srcMemory[1]->getDataAs<const uint8_t>();
+    uint8_t* dstData = dstMemory[0]->getDataAs<uint8_t>();
+    cpu_memcpy(dstData, srcData, srcMemory[0]->getSize());
+    if (srcMemory[1]->getSize() == 0) {
+        // Updates are empty - do not apply
+        return;
+    }
+    uint8_t* dstShiftedData = dstData + srcShift;
+    parallel_nt(nThreads, [&](const int ithr, const int nthr) {
+        size_t start = 0, end = 0;
+        splitter(workAmount, nthr, ithr, start, end);
+
+        for (size_t iwork = start; iwork < end; ++iwork)
+            cpu_memcpy(&dstShiftedData[srcIndices[iwork]], &srcUpdates[dstIndices[iwork]], lastDstDim);
+    });
```

Comment:
The `lastDstDim` can have different meaning for SliceScatter and StridedSlice, please confirm that it is applicable for all cases of SliceScatter.

Reply:
As far as I've checked only problem with lastDstDim was with blocked mem layout where updates padding was included, however blocked was disabled. Other than that I couldn't find any other issues with it for now.
Like with possible issues with shapeHasDataDependency and missing execPtr, I can doublecheck it in separate ticket